.all-contributorsrc

```diff
@@ -22,6 +22,15 @@
       "contributions": [
         "translation"
       ]
+    },
+    {
+      "login": "carver",
+      "name": "Jason Carver",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/205327?v=4",
+      "profile": "http://www.linkedin.com/in/cryptocarver",
```

Comment:
If we only get one link, then I'd prefer https://github.com/carver

Reply:
👌 will update. FYI the bot pulls it from your GH profile

Reply:
Hah, okay, when I was picking that link, it seemed unnecessarily self-referential to link out to my github page from... inside my github page.